script.js

```diff
@@ -1,4 +1,118 @@
+var playerOne = "Player 1";
+var playerTwo = "Player 2";
+var gameMode = "Player 1's no choice yet";
+var p1DiceOne = "";
+var p1DiceTwo = "";
+var p2DiceOne = "";
+var p2DiceTwo = "";
+var pOneOrderA = [];
+var pOneOrderB = [];
+var pTwoOrderA = [];
+var pTwoOrderB = [];
+var conp1num = "";
+var conp2num = "";
+var p1num = "";
+var p2num = "";
+
+//Helper function : the Dice Rolls:
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+//when player 1 click submit, the game rolls 2 dice
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var OutputValue = "";
+
+  var playerOneChoice = [];
+  if (gameMode == "Player 1's no choice yet") {
+    //roll the 2 dice
+    var diceOne = diceRoll();
+    var diceTwo = diceRoll();
+    //and shows the 2 dice roll result
+    var myOutputValue = `Welcome ${playerOne}. <br>You rolled ${diceOne} for Dice 1 and ${diceTwo} for Dice 2. <br>Choose the order of the dice.`;
+
+    //capture the dice rolls as variables and update them for global usage later
+    p1DiceOne = diceOne;
+    p1DiceTwo = diceTwo;
+
+    //update the "game mode" to "player 1 going to make a choice"
+    gameMode = "Player 1 makes choice";
+  } else if (gameMode == "Player 1 makes choice") {
+    var playerOneChoice = input;
+
+    //Order of the dice - captured as an array:
+    var pOneOrderA = [p1DiceOne, p1DiceTwo];
```

Comment:
I like how you're storing the dice in an array, but I don't think this array is used anywhere in your code after you declared it.